bosh-setup/scripts/setup_env

```diff
@@ -57,6 +57,27 @@ sed -i "s/CLOUD_FOUNDRY_PUBLIC_IP/cf-ip/g" settings
 cp settings $home_dir
 
 sed -i "s/REPLACE_WITH_BOSH_DIRECOT_IP/$bosh_director_ip/g" deploy_bosh.sh
+
+REPLACE_WITH_REGISTRY_HTTP_PASSWORD=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)
```

Comment:
@haydonryan 
From my test, the script hangs in `cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1`.
Could you reproduce the hanging by deploying the template with your PR?